floppy/upgrade-wmf.bat

```diff
@@ -0,0 +1,80 @@
+:::::::
+:: Script pulled from:
+:: https://blogs.technet.microsoft.com/pstips/2015/04/11/powershell-4-0-deployment/
+:::::::
+@echo off
+title %~nx0
+cls
+
+
+set "outputFolder=%TEMP%"
+if /i not exist "%outputFolder%" md "%outputFolder%"
+>> "%outputFolder%\%~n0.log" 2>&1 (
+call :START ) & endlocal & goto:eof
+
+
+:START
+echo %date% %time% - %~nx0 started
+if /i "%PROCESSOR_ARCHITECTURE%"=="x86" (
+    if not defined PROCESSOR_ARCHITEW6432 (
+        set BITNESS=x86
+    ) else (
+        set BITNESS=x64
+    )
+) else (
+    set BITNESS=x64
+)
+
+
+:NETFX45
+reg query "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\.NETFramework\v4.0.30319\SKUs\.NETFramework,Version=v4.5" 2>NUL
+if %ERRORLEVEL% equ 0 (
+    echo .NET Framework 4.5 is already installed
+    goto WMF4
+)
+echo Downloading .NET Framework 4.5 Offline Installer...
+powershell.exe -File A:\wget.ps1 https://download.microsoft.com/download/E/2/1/E21644B5-2DF2-47C2-91BD-63C560427900/NDP452-KB2901907-x86-x64-AllOS-ENU.exe %outputFolder%\NDP452-KB2901907-x86-x64-AllOS-ENU.exe
+
+echo Installing .NET Framework 4.5
+start /wait %outputFolder%\NDP452-KB2901907-x86-x64-AllOS-ENU.exe /q /log %outputFolder%\netfx45.htm /norestart%
+if %ERRORLEVEL% equ 0 goto WMF4
+if %ERRORLEVEL% equ 1641 goto WMF4
```

Comment:
Incase anyone else is wonder these error codes mean `ERROR_SUCCESS_REBOOT_INITIATED` and `ERROR_SUCCESS_REBOOT_REQUIRED`.